src/main/java/core/basesyntax/service/impl/DeepCopyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package statement to follow Java conventions.